recipes/mintie/mintie-wrapper.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+MINTIELOGO=$(cat <<'END'
+  __  __ ___ _   _ _____ ___ _____
+ |  \/  |_ _| \ | |_   _|_ _| ____|
+ | |\/| || ||  \| | | |  | ||  _|
+ | |  | || || |\  | | |  | || |___
+ |_|  |_|___|_| \_| |_| |___|_____|
+
+Method for Inferring Novel Transcripts and Isoforms using Equivalences classes
+END
+)
+
+downgrade_soap() {
```

Comment:
Have you considered adding a recipe for soapdenovo-trans 1.03? It’s a bit of extra work, but it would be a much less ugly hack.

Reply:
Yes, this would be ideal, however I'm not sure how to add an older version of a tool already on bioconda. Would this be added as a new recipe? If so, how should I name the recipe folder, given that these don't contain version information?

Reply:
I found the thread about adding older recipes as subfolders. I'll give this a try.

Reply:
Perfect!